src/player.cpp

```diff
@@ -771,7 +767,7 @@ void Player::addStorageValue(const uint32_t key, const int32_t value, const bool
 		storageMap[key] = value;
 
 		if (!isLogin) {
-			int64_t currentFrameTime = OutputMessagePool::getInstance()->getFrameTime();
+			int64_t currentFrameTime = OTSYS_TIME();
```

Comment:
So I will need to add a dispatcher cycle counter, right? Using current time for the counter seems like an unnecessary overhead anyway.